andela/hc-wits-kla
# #158175317 Notify the User using SMS and Telegram

### What does this PR do?
- Ensure that the user is notified using other channels, like SMS and Telegram
### How should this be manually tested?
- Clone the repo, `git clone https://github.com/andela/hc-wits-kla`
- In your terminal, `cd hc-wits-kla`
- In the project directory `git checkout ft-sms-telegram-158175317`
- In the project directory, activate the environment, `source .env`
- Install requirements `pip install -r requirements.txt`
- Make migrations `python manage.py makemigrations`
- Run migrations `python manage.py migrate`
- Serve the application `python manage.py runserver`
- In the browser, visit, `127.0.0.1:8000/checks/`
- In your browser create a check, set **Period** to 2 minutes and **Grace Time** to 1 minute.
- Navigate to the `Integrations` tab and choose SMS and Telegram, and register your phone number and Telegram username on the corresponding integration pages.
- In your terminal, listen for alerts `python manage.py sendalerts`
- Ping the check you created earlier (First ping), and wait for more than 3 minutes.
- After the check goes down, a Telegram notification and an SMS notification about the check should be sent.
- Ping the check again (Second ping) to change its status to `up` and another notification will be sent on the check's status
### Relevant Pivotal Tracker story.
[#158175317](https://www.pivotaltracker.com/story/show/158175317)
